code/__DEFINES/ai/ai_blackboard.dm

```diff
@@ -67,6 +67,8 @@
 
 ///List of mobs who have damaged us
 #define BB_BASIC_MOB_RETALIATE_LIST "BB_basic_mob_shitlist"
+///do we currently have a retaliate target?
+#define BB_BASIC_MOB_CURRENTLY_RETALIATTING "BB_basic_mob_currently_retaliating"
```

Comment:
Maybe we could just call it `BASIC_MOB_IGNORE_FACTIONS` to be a bit more generic, you could have a different behaviour toggle this same blackboard key for some other reason (as long as you're careful not to have two behaviours compete with each other)